### DISCUSSION TITLE
支持接入腾讯云内容安全 || Support access to Tencent Cloud content security

### DISCUSSION THREAD

**Discussion by sysfox** [walinejs/waline#2766] [Ideas]
我注意到Waline已经支持了Akismet
那么能否让Waline支持接入如腾讯云内容安全服务，毕竟其在国内的访问速度更快？

---
I noticed that Waline already supports Akismet
So can Waline support access to content security services such as Tencent Cloud, after all, its access speed in China is faster?

**Comment by lizheming** [comment]
https://github.com/walinejs/plugins/tree/master/packages/tencent-tms
